react-client-app/src/components/Home.jsx

```diff
@@ -101,20 +101,34 @@ const Home = () => {
         // sort scan reports
         scanreports = scanreports.sort((b, a) => (a.id > b.id) ? 1 : ((b.id > a.id) ? -1 : 0))
         // create a list of unique datapartners and make a batch query to get their data
-        const dataPartnerObject = {}
+        const datasetObject = {};
         scanreports.map(scanreport => {
-            dataPartnerObject[scanreport.data_partner] = true
+            datasetObject[scanreport.parent_dataset] = true;
         })
-        const dataPartnerIds = chunkIds(Object.keys(dataPartnerObject))
-        const dataPartnerPromises = []
+        const datasetIds = Object.keys(datasetObject)
+        const datasetPromises = [];
+        for (let i = 0; i < datasetIds.length; i++) {
+            datasetPromises.push(useGet(`/datasets/${datasetIds[i]}`));
```

Comment:
is this because the api endpoint is for individual datasetIDs rather than being able to do one api call with something like:
```
/datasets/?id__in=${dataSetIds[i].join()
```
?

I guess it's unlikely for datasetIds to be a long array?  And the code won't be hammering the api with lots of requests?

Reply:
Yeah it's because the endpoint "/datasets/?id__in= " doesn't exist yet. I need to create it first

Reply:
yeh sorry just seen your comment on the main conversation now that they need to be added